app/api.py

```diff
@@ -130,5 +133,50 @@ def post_reasons_for_contacting(self, form=None, payload=None):
         payload = form.api_payload() if form else payload
         return self.post("checker/api/v1/reasons_for_contacting/", json=payload)
 
+    def format_slots_by_day(self, slots):
+        slots_by_day = {}
+
+        for slot in slots:
+            date_str = slot.date().strftime("%Y-%m-%d")
+
+            if date_str not in slots_by_day:
```

Comment:
you can also replae the if condition and it's body with `slots_by_day.setdefault(date_str, [])`